source/mir/math/internal/logBeta.d

```diff
@@ -0,0 +1,32 @@
+/++
+License: $(HTTP www.apache.org/licenses/LICENSE-2.0, Apache-2.0)
+
+Authors: John Michael Hall
+
+Copyright: 2022 Mir Stat Authors.
++/
+
+module mir.math.internal.logBeta;
```

Comment:
`log_beta` please

Reply:
@jmh530 Sorry, I haven't been clear enough. The `logBeta` name for functions is gooid. `log_beta`  is for module only.